WalletWasabi.Documentation/Guides/InstallInstructions.md

```diff
@@ -171,11 +174,11 @@ If you prefer the Graphical User Interface, this guide is yours. There is also a
 
 4. Save the file and close.
 
-5. Right click on pgp.txt. In the context menu navigate to More GpgEx options/Import keys. If the context menu is missing -  restart the computer.
+5. Right click on pgp.txt file. In the context menu navigate to More GpgEx options and click Import keys. If the context menu is missing restart the computer.
```

Comment:
Nit:

feels like `missing` and `restart` should be separated at least by a comma, so hyphen feels appropriate, but I'm not a punctuation expert.